### ISSUE TITLE
Is this repository still necessary?

### ISSUE THREAD

**Issue by smashwilson** [deconst/map#1]
I like the idea of using [map-example](https://github.com/deconst/map-example) as the "development" map repository, and as a working sample for anyone else who wants to run a deconst-powered site. I feel like the "real" map repository for our development site belongs in rackerlabs, though.

**Comment by kenperkins** [comment]
the mapping service, as I understand it, is deployment agnostic, but the mapping service instance is specific a deployment?

**Comment by smashwilson** [comment]
When you start a mapping service, you'll provide the address of the map repository (or map repositories?) for that instance. It's part of the system configuration. In a lot of ways, the map repository defines the site: it controls which content repositories are included and where.

If I recall correctly, we started this one to be the "real" map repository for the site that hosts developer.rackspace.com, etc., but now that I think about it I feel like that doesn't belong here. The deconst org is for the generic platform: specific deployments of it can be launched pointing to content and map repositories wherever.

**Comment by kenperkins** [comment]
So I think we need a `mapping-service` repo, and a `map-example`?

**Comment by smashwilson** [comment]
Yes, that's what I'm getting at. See also: deconst/path2source#1. I'll handle it, but I want to coordinate with @rgbkrk before I delete and rename a bunch of stuff that he created.
